media.info.yml

```diff
@@ -6,4 +6,5 @@ core: 8.x
 dependencies:
   - media_entity
   - media_entity_image
-
+  - video_embed_field
+  - video_embed_media
```

Comment:
The other day I learned that we support namespacing of dependencies. See how we did it in media entity: https://www.drupal.org/node/2744455

Also see relevant change record: https://www.drupal.org/node/2299747

This might also solve the problem that required  before_script.sh. Not sure, but you should try it out.